internal/install/application_configuration_test.go

```diff
@@ -52,8 +53,12 @@ func TestSelectElasticAgentImageName_NextStackInOwnNamespace(t *testing.T) {
 	assert.Equal(t, selected, elasticAgentCompleteImageName)
 }
 
-func TestSelectElasticAgentImageName_DefaultImage816(t *testing.T) {
+func TestSelectElasticAgentImageName_DefaultImage816_WihtoutEnvVar(t *testing.T) {
 	version := stackVersion8160
+	// Try to keep the test agnostic from the environment variables defined in CI
+	t.Setenv(disableElasticAgentWolfiEnvVar, "")
+	os.Unsetenv(disableElasticAgentWolfiEnvVar)
```

Comment:
Could this cause any issues ?
It looks like the environment variable is defined in other tests even here it is unset.

Reply:
Setting environment variables is only going to be problematic if tests are run in parallel, as it affects the global process. `t.Setenv` recovers the previous value on cleanup, see https://pkg.go.dev/testing#B.Setenv.

Reply:
Perfect, so as these tests do not use `t.Parallel()`, I guess it is safe here. Even if it is called `os.Unsetenv` , the environment variable would be recovered aftwards thanks to the t.Setenv()

Reply:
I think each package is executed in a different process, so we should be fine if the tests are in different packages.